From ed2be6056f1a63f70c5bb561b80d8b87900dad32 Mon Sep 17 00:00:00 2001
From: Jerry Jelinek <jerry.jelinek@joyent.com>
Date: Wed, 14 Jun 2017 13:59:54 +0000
Subject: [PATCH] OS-6187 merge lost inotify man pages Reviewed by: Patrick
 Mooney <patrick.mooney@joyent.com> Reviewed by: Dan McDonald
 <danmcd@joyent.com> Approved by: Patrick Mooney <patrick.mooney@joyent.com>

---
 usr/src/man/man3c/Makefile | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/usr/src/man/man3c/Makefile b/usr/src/man/man3c/Makefile
index 0d8c89c217..fc3bcc8bb6 100644
--- a/usr/src/man/man3c/Makefile
+++ b/usr/src/man/man3c/Makefile
@@ -211,6 +211,9 @@ MANFILES=	__fbufsize.3c					\
 		imaxdiv.3c					\
 		index.3c					\
 		initgroups.3c					\
+		inotify_init.3c					\
+		inotify_add_watch.3c				\
+		inotify_rm_watch.3c				\
 		insque.3c					\
 		is_system_labeled.3c				\
 		isaexec.3c					\
-- 
2.21.0

